app/features/config/index.js

```diff
@@ -28,5 +28,26 @@ export default {
     /**
      * The URL of Terms and Conditions Page.
      */
-    termsAndConditionsURL: 'https://jitsi.org/meet/terms'
+    termsAndConditionsURL: 'https://jitsi.org/meet/terms',
+
+    /**
+     * Some of default values for application.
+     */
+    defaults: {
+        alwaysOnTopWindowEnabled: true
+    },
+
+    /**
+     * All needed configurations for storage in our application
+     * This is including electron-store and redux-persist
```

Comment:
I don't want to have 2 ways to persist storage. Let's exist with the existing redux-persist please.

Reply:
I don't understand comment, you need to use electron-store to access redux-persist data from main.js
And redux-persists uses electron-store to read/write data on user device

Reply:
Oh, I'll check it again then.